e2e-tests/src/page-components/navbar.ts

```diff
@@ -0,0 +1,34 @@
+import { Locator, Page } from '@playwright/test'
+import ComponentBase from 'src/models/component-base'
+
+/* navbar is located at the top of OurHealth page */
```

Comment:
Good to know. An easy change since we have few abstract classes. 

* Removed `navbar` object from `src/models/page-base.ts` abstract class.
* Created a new `pages/ourhealth/ourhealth-page-base.ts` abstract class `extends PageBase` with only `navbar` object.

```
import { Page } from '@playwright/test'
import PageBase from 'src/models/page-base'
import Navbar from 'src/page-components/navbar'

export default abstract class OurhealthPageBase extends PageBase {
  title = 'OurHealth'

  navbar: Navbar

  protected constructor(page: Page) {
    super(page)
    this.navbar = new Navbar(page)
  }
}
```